common/tools.py

```diff
@@ -389,7 +389,8 @@ def get_git_repository_info(path=None, hash_length=None):
     """
 
     if not path:
-        path = pathlib.Path.cwd()
+        # This should be the repositories root folder
```

Comment:
This function did not worked as expected in my manual tests. Not sure if it was the case in the past or if this is because of my wrong-solved merge conflicts.